src/components/modal/Modal.scss

```diff
@@ -153,6 +157,15 @@
         .modal-close-button {
             right: 10px;
         }
+
+        .modal-actions {
+            position: fixed;
+            right: 0;
+            bottom: 0;
+            width: 100%;
+            padding: 2.5 * $bdl-grid-unit 4 * $bdl-grid-unit;
```

Comment:
Let's use the numeric values here, too, since they're basically copied from other styles in this file.